src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -84,6 +84,10 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
         };
 
         $scope.editNoteSwitch = function() {
+            if ($scope.isRecruiter === false && $scope.relativeDayNumber < 0) {
```

Comment:
This conditional expression is duplicated within the controller -- please extract it to a separate function and use that function instead.